101-servicebus-create-namespace-geo-recoveryconfiguration/azuredeploy.json

```diff
@@ -0,0 +1,82 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2014-04-01-preview/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "serviceBusNamespaceNamePrimary": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of Service Bus namespace"
+      }
+    },
+    "serviceBusNamespaceNameSecondary": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of Service Bus namespace"
+      }
+    },
+    "aliasName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of Geo-Recovery Configuration Alias "
+      }
+    }
+  },
+  "variables": {
+    "defaultSASKeyName": "RootManageSharedAccessKey",
+    "defaultAuthRuleResourceId": "[resourceId('Microsoft.ServiceBus/namespaces/authorizationRules', parameters('serviceBusNamespaceNamePrimary'), variables('defaultSASKeyName'))]"
+  },
+  "resources": [
+    {
+      "apiVersion": "2017-04-01",
+      "name": "[parameters('serviceBusNamespaceNameSecondary')]",
+      "type": "Microsoft.ServiceBus/Namespaces",
+      "location": "North Central US",
```

Comment:
don't hard code the locations - since this is relevant to the geo-dist of the sample

Reply:
namespaces should be in 2 different regions, for one namespace have used the resource group location and hard coded the second. please let me know if we can assign a different region

Reply:
use parameters or variables